lib/foreman_docker/permitted_attributes.rb

```diff
@@ -0,0 +1,10 @@
+module Foreman
+  module PermittedAttributes
+    mattr_reader :container_attributes
+
+    @@container_attributes = [:command, :repository_name, :name, :compute_resource_id, :entrypoint,
```

Comment:
this PR is obsolete and was written for a previous iteration on the rails 4 branch, which is no longer in existence.